### PR TITLE
models.yml for vote-decrypt

### PR DESCRIPTION
**Pull request by ostcar**
This is only an example. Especially the names could be different.

### CODE REVIEW

File: global/meta/models.yml [hunk 1]
```diff
@@ -2700,6 +2700,7 @@ poll:
       - analog
       - named
       - pseudoanonymous
+      - crypt
```
Comment (jsangmeister) [inline global/meta/models.yml:2703]: I think I would favor "anonymous". If you disagree, I would at least write the full word ("cryptographic") to stay in lane with the other names.

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: global/meta/models.yml (hunk 1 above)
Comment (ostcar) [reply global/meta/models.yml:2703]: I have no preference. 

anonymous sounds like the poll is for anonymous users.

The English term for "geheime Wahl" is "secret ballot". So another option would be "secret".

Comment (jsangmeister) [reply global/meta/models.yml:2703]: That is right, also we already have a similar "problem" with "pseudoanonymous". But "cryptographic" but be better suited under these circumstances.

Comment (ostcar) [reply global/meta/models.yml]: done

File: global/meta/models.yml [hunk 2]
```diff
@@ -2806,6 +2807,26 @@ poll:
     read_only: true
     required: true
     restriction_mode: A
+  cryptographic_key:
+    type: string
```
Comment (r-peschke) [inline global/meta/models.yml:2811]: I suppose this and also following names are not visible for the customer, then I would prefer shorter names: `crypt_key`

File: global/meta/models.yml [hunk 3]
```diff
@@ -2806,6 +2807,26 @@ poll:
...
+    description: base64 public key to cryptographic votes.
+    read_only: true
+    restriction_mode: A
+  cryptographic_key_signature:
```
Comment (r-peschke) [inline global/meta/models.yml:2815]: `crypt_signature`, analogue to votes_raw and votes_signature

**Review by r-peschke — CHANGES_REQUESTED** [review]
My proposal to use shorter names, if you both agree

File: global/meta/models.yml (hunk 2 above)
Comment (ostcar) [reply global/meta/models.yml:2811]: The keys are not visible. My first proposal was also "crypt_key". I changed this to have it consistent with the value of `poll/type`. @jsangmeister asked to change the value of `poll/type`. See the outdated discussion above.

File: global/meta/models.yml (hunk 3 above)
Comment (ostcar) [reply global/meta/models.yml:2815]: I don't see the analogy to `votes_raw` and `votes_signature`. What are your proposals for the names of these keys?

**Comment by ostcar** [comment]
> My proposal to use shorter names, if you both agree

I have no preference. @r-peschke and @jsangmeister please discuss this among each other and change the PR afterwards.

File: global/meta/models.yml (hunk 3 above)
Comment (r-peschke) [reply global/meta/models.yml:2815]: votes_raw and votes_signature, but crypt_key and crypt_key_signature. In one case you strip the "usage" (raw), in the other case you leave the "usage" (key).

**Review by r-peschke — APPROVED** [review]

**Comment by r-peschke** [comment]
@jsangmeister Fell free to change the names, if it is important for you

File: global/meta/models.yml (hunk 2 above)
Comment (jsangmeister) [reply global/meta/models.yml:2811]: I have no strong opinion on the name of these fields, so we can leave it as it currently is

**Pull request merged**